### PR TITLE
Update custom domain doc with more cert details

### PR DESCRIPTION
**Pull request by dlepow**

### CODE REVIEW

**Comment by dlepow** [comment]
@avtakkar @toddysm - Please review this doc update for custom domains. Thank you!

File: docs/custom-domain/README.md [hunk 1]
```diff
@@ -91,6 +106,12 @@ For each domain, its TLS private key and public certificate pair must be added t
 
 For greater isolation, we recommend that you put each certificate in its own key vault and set its access policy independently. The registry should always have access to the key vault secrets.
 
+### Certificate updates and rotation
+
+The registry configures each custom domain certificate without referencing a specific version in Azure Key Vault. With this configuration, the registry regularly checks the key vault and automatically uses the latest certificate version there for its operations.
```
Comment (aviral26) [inline docs/custom-domain/README.md:111]: This is configurable. Customers have the option to use a versioned secret ID as well if they do not want auto cert rotation.

Comment (dlepow) [reply docs/custom-domain/README.md]: Updated - thanks @aviral26

**Comment by dlepow** [comment]
@sajayantony - Can you please merge this PR if it appears ready to you? Many thanks.

**Review by sajayantony — APPROVED** [review]

**Pull request merged**